fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -2340,6 +2351,525 @@ countries:
+                chirps-precip-jjaso:
+                    label: CHIRPS Precip Jun-Oct
+                    description: CHIRPS total precipitation for Jun-Oct
+                    path: mauritania/chirps-precip-jjaso.zarr
+                    var_names:
+                        value: precipitation
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                chirps-precip-jas:
+                    label: CHIRPS Precip JAS
+                    description: CHIRPS total precipitation for Jul-Sep
+                    path: mauritania/chirps-precip-jas.zarr
+                    var_names:
+                        value: precipitation
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                chirp-spi-jj:
+                    label: CHIRP SPI Jun-Jul
+                    description: Average SPI in June-July, calculated from CHIRP
+                    path: mauritania/chirp-spi-jj-jas.zarr
+                    var_names:
+                        value: spi
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                    format: number3
+                chirp-spi-jas:
+                    label: CHIRP SPI JAS
+                    description: Average SPI in July-September, calculated from CHIRP
+                    path: mauritania/chirp-spi-jas.zarr
+                    var_names:
+                        value: spi
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                    format: number3
+                chirp-rainfall-sep:
+                    label: CHIRP Rain Sep
+                    description: Total rainfall in September according to CHIRP
+                    path: mauritania/chirp-rainfall-sep-jas.zarr
+                    var_names:
+                        value: prcp
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                    format: number3
+#                chirps-dryspell:
+#                    label: Dry Spell
+#                    description: Longest dry spell, calculated from CHIRPS
+#                    path: mauritania/chirps-dryspell-jas.zarr
+#                    var_names:
+#                        value: lls
+#                        lat: Y
+#                        lon: X
+#                        time: T
+#                    units: days
+#                    colormap: precip
+#                    lower_is_worse: no
+#                    format: number2
```

Comment:
I tried uncommenting this and it seems to work fine. Are you still having a problem with it?